mapping/sustainability.yaml

```
@@ -1143,8 +1189,52 @@
     module: Social
     indicator: Beneficiary population
     disclosure format: 'Indicate the number of direct and indirect project beneficiaries (E.g. direct: \[number\]; indirect: "number"). Beneficiaries are the individuals who benefit directly or indirectly from the project; they are the target group of the infrastructure project and their needs are addressed by the intervention.'
-    mapping: ''
-    example: ''
+    mapping: |-
+        Project level: Add a `benefit` object to the `benefits` array.
+
+        For direct beneficiaries:
+
+        - Add a `Beneficiary` object to the `.beneficiaries` array.
+        - Set the `.description` to "direct beneficiaries".
+        - Set the number of people to `.numberOfPeople`.
+        - Add the location of the beneficiary population to `.location`
+
+        For indirect beneficiaries:
+
+        - Add a `Beneficiary` object to the `.beneficiaries` array.
+        - Set the `.description` to "indirect beneficiaries".
+        - Set the number of people to `.numberOfPeople`.
+        - Add the location of the beneficiary population to `.location`
```

Copy-edit. Capitalise definitions. Remove reference to `.location` since it isn't mentioned in the disclosure format.

```suggestion
        Project level:
        
        1. Add a `Benefit` object to the `benefits` array.
        2. Add a `Beneficiary` object to the benefit's `.beneficiaries` array, set its `.description` to "Direct beneficiaries" and set its `.numberOfPeople` to the number of direct beneficiaries.
        3. Add a `Beneficiary` object to the benefit's `.beneficiaries` array, set its `.description` to "Indirect beneficiaries" and set its `.numberOfPeople` to the number of indirect beneficiaries.
```